### DISCUSSION TITLE
Can we add checkbox, buttons and custom right click commands to ttkbootstrap Tableview?

### DISCUSSION THREAD

**Discussion by sunny9495-dev** [israel-dryer/ttkbootstrap#256] [Q&A]
HI @israel-dryer 

I hope you have a great day, just want to ask is it possible to add checkbox, button and custom right click to ttkbootstrap Tableview.

If yes, can you please give a example.

Thanks

**Comment by israel-dryer** [comment]
Because of how the Tableview is designed, this is not really possible. It is kind-of possible with the Treeview with a hack, but it's not really intended to be used that way.

**Comment by sunny9495-dev** [comment]
Ok @israel-dryer Thanks
